NewChallenge2.js

```diff
@@ -0,0 +1,89 @@
+// CHALLENGE 2
+
+// Make a program where you ask the user to input a chore (so use prompt again)
+// It must then randomly choose someone from an array and print that they need to do the chore
+// So you'd ask "What chore needs to be done?" -> and then type dishes as an example
+// and it would console.log "Jason needs to do dishes today"
+
+let chore = document.querySelector(".chore");
+let name = document.querySelector(".name");
+
+let choreBtn = document.querySelector(".choreBtn");
+
+choreBtn.addEventListener("click", (e) => {
+
+  return (addToArray(chore, e, choreArray, chore.value, '#chores-list'))
+});
+
+let choreArray = [];
```

Comment:
avoid naming with the type (Array) Rather name this `chores` - the plural indicates a collection of chores